lib/common/Job.ts

```diff
@@ -1,7 +1,4 @@
-export enum JobStatus {
-  RUNNING = 'running',
-  STOPPED = 'stopped',
-}
+export type JobStatus = 'running' | 'stopped'
```

Comment:
I'd prefer to support both approach, export enum and make it possible to pass value by string.
Can you adjust type of `JobStatus` to be values of JobStatusEnum, and export both?

Reply:
Sorry, I'm not sure I understood what you mean correctly, but could you view my second commit (e97f435)? Is it like that?

Reply:
@kibertoad Hi, are you here? :)

Reply:
yup, will do a little tweak and merge today